x/oracle/client/cli/query.go

```diff
@@ -0,0 +1,61 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	"github.com/provenance-io/provenance/x/oracle/types"
+)
+
+// GetQueryCmd is the top-level command for oracle CLI queries.
+func GetQueryCmd() *cobra.Command {
+	queryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the oracle module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	queryCmd.AddCommand(
+		GetQueryOracleAddressCmd(),
+	)
```

Comment:
Missing a command for the `Oracle` query.

Reply:
I opted not to expose it since it's the same as doing a wasm query. The only reason I have a separate message for it is to make querying the oracle easier from another chain. The user doesn't need to look up an address on a remote chain.